src/pages/dashboard/row.svelte

```diff
@@ -181,6 +181,12 @@
                     color: var(--main-black);
                 }
             }
+            .send {
+                padding: 9px;
+                :global(.button) {
```

Comment:
why are you adding this styling globally? 🤔

Reply:
I corrected it in the wrong place, I should set padding in the max-width:600 block, and remove the 
globally style。I commit again